### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by addod19**
pull request for code review

### CODE REVIEW

**Comment by HenryTabima** [comment]
# Great Job! :100: 
Your Readme file is really impressive :1st_place_medal:. Your code is clean and well organized. Just a few minor changes to give you a thumbs up.

## Top
- [ ] It has a title `Create an Intuit account`.

#### Happy coding :computer:

File: index.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    <main class="main">
+        <div class="products">
+            <img class="all" src="./assets/svg/all.png" alt="All logo" />
+        </div>
+        <!--beg form container-->
+        <div class="form-container">
+
+            <!-- begin form header-->
+            <div class="form-header">
+                <span class="form-title">Create an Intuit account</span>
```
Comment (HenryTabima) [inline index.html]: this should be a title (`<h#>` tag) as `h1` or `h3`, the one you prefer.

**Review by bellom — APPROVED** [review]
Hi @addod19 
Great work, your code is error free and well structured.
## Congratulation :+1:

**Comment by addod19** [comment]
thank you very much.

**Comment by addod19** [comment]
merging dev to master

**Pull request merged**